src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -299,7 +301,7 @@ MultiPlanTree(Query *queryTree)
 		 * elements.
 		 */
 		joinClauseList = JoinClauseList(whereClauseList);
-		tableEntryList = TableEntryList(rangeTableList);
+		tableEntryList = UsedTableEntryList(queryTree);
```

Comment:
There's other places that use TableEntryList - why don't they need to be adjusted?